jerry-core/parser/js/js-parser-expr.c

```diff
@@ -2434,6 +2434,13 @@ parser_pattern_process_assignment (parser_context_t *context_p, /**< context */
       {
         parser_raise_error (context_p, PARSER_ERR_INVALID_DESTRUCTURING_PATTERN);
       }
+      else if (context_p->last_cbc_opcode == CBC_PUSH_PROP_THIS_LITERAL)
```

Comment:
There is no need else here if I understand the code correctly.

Reply:
Indeed, fixed.